src/msxcompiler.c

```diff
@@ -232,22 +235,14 @@ void  writeSrcFile(FILE* f)
 "   #define DLLEXPORT \n"
 " #endif \n"
 "  \n"
-" void  DLLEXPORT  MSXgetPipeRates(double c[], double k[], double p[], double h[], double f[], int TheLink, double t); \n"

```

Comment:
These changes seem to be problematic. Clang picks up on the difference in function signatures.

```
epanet-msx/src/msxchem.c:921:73: error: too many arguments to function call, expected 4, have 5
            MSXgetPipeFormulas(ChemC1, MSX.K, MSX.Link[TheLink].param, HydVar, TheLink);
            ~~~~~~~~~~~~~~~~~~                                                 ^~~~~~~
```